modules/standard/CommDiagnostics.chpl

```diff
@@ -215,6 +215,53 @@ module CommDiagnostics
       non-blocking remote executions
      */
     var execute_on_nb: uint(64);
+
+    proc writeThis(c) {
```

Comment:
This is fine for now, but could probably be more elegantly written using Reflection.

Reply:
I'm not understanding this comment...

Reply:
I think you could use numFields, getFieldName, and getField within a param loop instead of many if-statements.